lib/motions/general-motions.coffee

```diff
@@ -140,9 +140,9 @@ class MoveLeft extends Motion
   operatesInclusively: false
 
   moveCursor: (cursor, count=1) ->
-    _.times count, ->
-      unless cursor.isAtBeginningOfLine()
-        cursor.moveLeft()
+    _.times count, =>
+      cursor.moveLeft() unless cursor.isAtBeginningOfLine() and !atom.config.get('vim-mode.wrapLeftRightMotion')
```

Comment:
Sorry, I'm not sure I can do that. I could make it 
```
      cursor.moveLeft() if atom.config.get('vim-mode.wrapLeftRightMotion') or !cursor.isAtBeginningOfLine()
or
      cursor.moveLeft() if !cursor.isAtBeginningOfLine() or atom.config.get('vim-mode.wrapLeftRightMotion')
```
But that's not really better, I suspect. What do you think?